ms-common-auth/src/main/java/net/trajano/ms/auth/internal/TokenCacheEntry.java

```diff
@@ -0,0 +1,81 @@
+package net.trajano.ms.auth.internal;
+
+import java.io.Serializable;
+import java.time.Duration;
+import java.time.Instant;
+
+/**
+ * Entries stored inside the cache.  The JWT is stored instead of a JwtClaims to allow encrypted JWTs to be used.  Only some meta data needs to be exposed.
+ */
+public class TokenCacheEntry implements Serializable {
+
+    private static final long serialVersionUID = -1643672416756418254L;
+
+    private final String accessToken;
+    private final String refreshToken;
+    private final String jwt;
+    private final String clientId;
+    private final Instant createdOn;
+    private final Instant expiresOn;
+
+    public TokenCacheEntry(String accessToken, String refreshToken, String jwt, String clientId) {
+        this(accessToken, refreshToken, jwt, clientId, null);
+    }
+
+    public TokenCacheEntry(String accessToken, String refreshToken, String jwt, String clientId, Instant expiresOn) {
+        this.accessToken = accessToken;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this value-based field transient so it is not included in the serialization of this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3437)